origin-discovery/src/listener/handler_identity.js

```diff
@@ -258,6 +260,10 @@ class IdentityEventHandler {
       user.profile.avatar = user.profile.avatar.slice(0, 32) + '...'
     }
 
+    if (log.decoded.ipfsHash) {
```

Comment:
Why is this condition necessary ? Is this to guard for future when this handler receives delete event that contrary to add/update event do not have an IPFS hash ?
https://github.com/OriginProtocol/origin/blob/master/origin-contracts/contracts/identity/IdentityEvents.sol#L17

If yes, maybe add a simple comment. It was not totally obvious to me at first.

Reply:
Mostly because I was being lazy and the tests didn't pass a full log so this failed. I'll add it to the tests. :grin: